assembly/Bool.ts

```diff
@@ -17,10 +17,14 @@ import { Codec } from "./interfaces/Codec";
 /** Representation for a boolean value in the system. */
 export class Bool implements Codec {
```

Comment:
Shouldn't we use `this._value` instead of `this.value` in the methods? If they are not throwing we can use `this.value` in every method (`populateFromBytes` currently uses `_value`)

Reply:
because `this.value` is getter, I am using `this._value` where I set the value and where I get it I use just `this.value`